tests/deit/console/option/parser/ArgvParserTest.php

```diff
@@ -11,14 +11,14 @@ class ArgvParserTest extends \PHPUnit_Framework_TestCase {
 
 	public function test() {
 
-		$console    = new GenericConsole();
+		$console    = new Event();
 		$argv       = array(
-			'my-cmd.php',
-			'-abc',
-			'-d=v',
-			'-test',
-			'--opt=test',
-		);
+							'my-cmd.php',
```

Comment:
Apologies for these.  Your preference for tabs isn't one I've 
encountered, so at this point it's mostly a manual conversion.  Will 
fix these and set up something to automate the process.

~gb